subgraph/src/public-lock.ts

```diff
@@ -341,13 +339,29 @@ export function handleRoleRevoked(event: RoleRevokedEvent): void {
       lock.keyGranters = newKeyGranters
       lock.save()
     }
+  } else if (
```

Comment:
added the logic with OZ role, as we were still using our custom events in the subgraph